lib/v2/douban/other/weekly_best.js

```diff
@@ -28,11 +28,13 @@ module.exports = async (ctx) => {
         link: `https://m.douban.com/subject_collection/${type}`,
         description: infoResponse.data.description,
 
-        item: data.map(({ title, info, cover, url, rating, year, release_date, null_rating_reason, description }) => {
+        item: data.map(({ title, info, cover, cover_url, url, rating, year, release_date, null_rating_reason, description }) => {
```

Comment:
```suggestion
        item: data.map(({ title, info, cover_url, url, rating, year, release_date, null_rating_reason, description }) => {
```

Reply:
the cover property is useful in the '`douban/movie/weekly/subject_real_time_hotest `' interface